csm/conf/etc/csm/csm.conf

```diff
@@ -158,3 +158,6 @@ MAINTENANCE:
 
 ELASTICSEARCH:
     retry: 5
+
+APPLIANCE_INFO:
+    serial_number_file_path: "/opt/seagate/lr-serial-number"
```

Comment:
csm should not bother where is the file. It should be taken care by cortx-utils code itself

Reply:
Done